modules/ROOT/pages/ibm/ibm-ctg-connector.adoc

```diff
@@ -455,7 +473,7 @@ values:
 }]
 ----
 +
-. Create a flowVar named request and configure the following fields (see section Load Java Metadata):
+. Create a flowVar named request and configure the following fields, as described in <<loadjavametadata>>:
```

Comment:
. Create a flowVar variable named `request` and configure the following fields, as described in <<loadjavametadata>>: